### PR TITLE
Button feature

### PR DESCRIPTION
**Pull request by mariyasada**
adding button feature to the component library. do review it

### CODE REVIEW

**Review by tushartiwari7 — APPROVED** [review]
I liked how you segmented the code.
Nice work!

File: htmlFiles/button.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+</section>
+</div>
+<!-- ---------------------------------------------------------javascript functionality------------------------------- -->
+<script>
+           const buttonsCodeBtn = document.querySelector("#buttons-code-btn");
+           const buttonsCodeDiv = document.querySelector("#buttons-div-code");
+
+           buttonsCodeBtn.addEventListener("click",() =>{
+                   console.log("clicked");
+                   buttonsCodeDiv .style.display =="block" ? buttonsCodeDiv .style.display ="none": buttonsCodeDiv .style.display ="block";
```
Comment (tusharkandpal) [inline htmlFiles/button.html]: What's this space here ?

**Review by tusharkandpal — COMMENTED** [review]
Superb work! 👍

File: stylesheets/button/button.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    display:flex;
+    flex-wrap: wrap;
+    max-width:80vw;
+    padding: 0.6rem;
+    gap:2.5rem;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    flex:1;
+}
```
Comment (JantuDeb) [inline stylesheets/button/button.css]: I would suggest that you should use a separate style sheet for documentation so, CSS for this container can be placed there.

**Review by JantuDeb — APPROVED** [review]
CSS for button component is looking good.

File: htmlFiles/button.html (hunk 1 above)
Comment (mariyasada) [reply htmlFiles/button.html]: the space in starting of click event?

File: stylesheets/button/button.css (hunk 2 above)
Comment (mariyasada) [reply stylesheets/button/button.css]: okay  i will do that

**Pull request merged**